src/v/redpanda/admin_server.cc

```diff
@@ -1045,19 +1045,51 @@ void config_multi_property_validation(
     if (updated_config.cloud_storage_enabled()) {
         // The properties that cloud_storage::configuration requires
         // to be set if cloud storage is enabled.
-        std::vector<std::reference_wrapper<
-          const config::property<std::optional<ss::sstring>>>>
-          properties{
-            std::ref(updated_config.cloud_storage_secret_key),
-            std::ref(updated_config.cloud_storage_access_key),
-            std::ref(updated_config.cloud_storage_region),
-            std::ref(updated_config.cloud_storage_bucket)};
+        using config_properties_seq = std::vector<std::reference_wrapper<
+          const config::property<std::optional<ss::sstring>>>>;
```

Comment:
Any suggestions on what are the best practices around local typedefs like this? 

I used this here as it is a somewhat complex type and we have two variables, but if this is overkill/not recommended I can remove it.

Reply:
I'm totally fine with doing this at function scope for brevity